lso/applications/objective_functions/squares/squares_area.py

```diff
@@ -0,0 +1,46 @@
+import numpy as np
+from typing import Optional
+
+from lso.utils import area_utils
+from lso.data import data as lso_data
+from lso.objective_function import objective_function as lso_objective_function
+
+
+class SquaresArea(lso_objective_function.ObjectiveFunction):
+
+    def __init__(self, shape_likeliness_coeff: Optional[float] = 1.0):
+        self.shape_likeliness_coeff = shape_likeliness_coeff
+
+    def calculate_shape_likeliness(self,
+                                   x: np.array,
+                                   metric: Optional[str] = 'Jaccard') -> np.array:
```

Comment:
comma at the end and close the bracket in the next line.

Reply:
done